src/js/components/LanguageWrapper.js

```diff
@@ -110,10 +110,12 @@ class LanguageWrapper extends Component {
   render() {
     const { lang } = this.state;
     const messages = localeMessages[lang];
+    // Right to Left Direction for arabic text
+    const direction = lang === 'ar' ? 'rtl' : 'ltr';
```

Comment:
can we make direction a property in js/i18n/constants.js file so we don't need inline conditionals. We can pull direction based on the preset constants.

Reply:
i think i know what you mean here. will do.